charts/sysdig/README.md

```diff
@@ -3,420 +3,330 @@
 ---
 **WARNING**
 
-This chart is being deprecated. Please use the [supported chart](https://github.com/sysdiglabs/charts/tree/master/charts/sysdig-deploy) for production deployments.
+This chart is being deprecated. Please use the [sydig-deploy chart](https://github.com/sysdiglabs/charts/tree/master/charts/sysdig-deploy) for production deployments.
 
 ---
 
+## Overview
 
-[Sysdig](https://sysdig.com/) is a unified platform for container and microservices monitoring, troubleshooting,
-security and forensics. Sysdig platform has been built on top of [Sysdig tool](https://sysdig.com/opensource/sysdig/)
-and [Sysdig Inspect](https://sysdig.com/blog/sysdig-inspect/) open-source technologies.
+`sysdig` chart adds the Sysdig installation components for [Sysdig Monitor](https://sysdig.com/product/monitor/) and [Sysdig Secure](https://sysdig.com/product/secure/) to all  the nodes in your Kubernetes cluster via a DaemonSet. This chart deploys the following Sysdig components into your Kubernetes cluster.
 
+- [Sysdig Agent](https://github.com/sysdiglabs/charts/tree/master/charts/agent)
+- [Node Analyzer](https://github.com/sysdiglabs/charts/tree/master/charts/node-analyzer)
+  - [Sysdig Benchmark Runner ](https://github.com/sysdiglabs/charts/tree/master/charts/node-analyzer#benchmark-runner)
+  - [Sysdig Host Analyzer](https://github.com/sysdiglabs/charts/tree/master/charts/node-analyzer#host-analyzer-legacy)
+  - [Sysdig Image Analyzer](https://github.com/sysdiglabs/charts/tree/master/charts/node-analyzer#node-image-analyzer)
+  - [Sysdig KSPM Analyzer](https://github.com/sysdiglabs/charts/tree/master/charts/node-analyzer#kspm-analyzer-preview)
+- [Sysdig KSPM Collector](https://github.com/sysdiglabs/charts/tree/master/charts/kspm-collector)
```

Comment:
I agree with @mavimo here and think that we should not be linking to the unrelated helm charts for these components here since the `sysdig` chart does not use them. Instead, we should either link to the relevant content at [https://docs.sysdig.com](https://docs.sysdig.com), or just not have these items be links.